src/dawn/engine_dawn.cc

```diff
@@ -534,7 +583,7 @@ Result EngineDawn::CreateFramebufferIfNeeded(
     render_pipeline->fb_size = size;
   }
   return {};
-}
+}  // namespace dawn
```

Comment:
I don't think this comment is correct? Doesn't this close the function?

Reply:
You're right!